docs/development/developer_guide.md

```diff
@@ -39,6 +39,7 @@ First, you need to run a Kubernetes cluster locally. There are lots of choices:
 
 - [local-up-cluster.sh in Kubernetes](https://github.com/kubernetes/kubernetes/blob/master/hack/local-up-cluster.sh)
 - [minikube](https://github.com/kubernetes/minikube)
```

Comment:
```suggestion
```

I'm curious about if we can remove these clusters. Actually, we verify the training-operator behavior using KinD in the CI.
@kubeflow/wg-training-leads WDYT?

Reply:
Yeah, I think we can target KinD as the main tool to develop Training Operator locally. Let's remove it

Reply:
@franciscojavierarceo Could you address my comments?

Reply:
Updated 👍